src/main/java/com/diamondfire/helpbot/bot/command/impl/other/ChannelMuteCommand.java

```diff
@@ -0,0 +1,87 @@
+package com.diamondfire.helpbot.bot.command.impl.other;
+
+import com.diamondfire.helpbot.bot.HelpBotInstance;
+import com.diamondfire.helpbot.bot.command.argument.ArgumentSet;
+import com.diamondfire.helpbot.bot.command.argument.impl.parsing.types.SingleArgumentContainer;
+import com.diamondfire.helpbot.bot.command.argument.impl.types.*;
+import com.diamondfire.helpbot.bot.command.help.*;
+import com.diamondfire.helpbot.bot.command.impl.Command;
+import com.diamondfire.helpbot.bot.command.permissions.Permission;
+import com.diamondfire.helpbot.bot.command.reply.PresetBuilder;
+import com.diamondfire.helpbot.bot.command.reply.feature.informative.*;
+import com.diamondfire.helpbot.bot.events.CommandEvent;
+import com.diamondfire.helpbot.sys.tasks.impl.MuteExpireTask;
+import com.diamondfire.helpbot.util.*;
+import net.dv8tion.jda.api.entities.*;
+
+import java.time.*;
+import java.util.Date;
+
+public class ChannelMuteCommand extends Command {
+    
+    @Override
+    public String getName() {
+        return "channelmute";
+    }
+    
+    @Override
+    public HelpContext getHelpContext() {
+        return new HelpContext()
+                .description("Mutes a player from posing in a specific channel for one week (7 days)")
+                .category(CommandCategory.OTHER)
+                .addArgument(new HelpContextArgument().name("user"))
+                .addArgument(new HelpContextArgument().name("channel"))
+                .addArgument(new HelpContextArgument().name("duration"))
+                .addArgument(new HelpContextArgument().name("reason").optional());
+    }
+    
+    @Override
+    protected ArgumentSet compileArguments() {
+        LocalDate nextWeek = LocalDate.now().plusDays(7);
+        return new ArgumentSet()
+                .addArgument("user", new DiscordUserArgument())
+                .addArgument("channel", new StringArgument())
+                .addArgument("duration", new SingleArgumentContainer<>(new TimeOffsetArgument()).optional(DateUtil.toDate(nextWeek)))
```

Comment:
Pass null for this. Arguments are cached, so this will not update. In execute method
``` java
if (duration == null) {
duration = DateUtil.toDate(LocalDate.now().plusDays(7));
}
```